src/test/librados/tier.cc

```diff
@@ -1220,6 +1220,154 @@ TEST_F(LibRadosTwoPoolsPP, EvictSnap2) {
   }
 }
 
+//this test case reproduces http://tracker.ceph.com/issues/17445
```

Comment:
@xxhdx1985126 I believe it would be good if we can start comment with Capital letter something as:
//This test case reproduces http://tracker.ceph.com/issues/17445